src/app/lib/config/services/kyrgyz.js

```diff
@@ -16,12 +16,12 @@ const service = {
     defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/kyrgyz.png',
     defaultImageAltText: 'BBC News Кыргыз КызMATы',
     dir: `ltr`,
-    externalLinkText: ', external',
+    externalLinkText: ', Би-Би-Сиден тышкары баракча',
     imageCaptionOffscreenText: 'Image caption, ',
```

Comment:
There are translations for these on the spreadsheet.

Reply:
Updated the translations that weren't updated before.